### PR TITLE
[Beta] Consume latest Azure Monitor OpenTelemetry

### PR DESCRIPTION
**Pull request by hectorhdzg**
Align with names used in Azure SDK projects
ApplicationInsightsClient to be used when manually using the SDK to not conflict with shim TelemetryClient
 Added LogApi class for manual ApplicationInsights event APIs

### CODE REVIEW

File: src/shim/applicationinsights.ts [hunk 1]
```diff
@@ -42,15 +31,7 @@ let _options: ApplicationInsightsOptions;
  * and start the SDK.
  */
 export function setup(setupString?: string) {
-    _setupCalled = true;
-    // Save the setup string and create a config to modify with other functions in this file
-    _setupString = setupString;
-    if (!_options) {
-        _options = { azureMonitorExporterConfig: { connectionString: _setupString } };
-    } else {
-        Logger.getInstance().info("Cannot run applicationinsights.setup() more than once.");
```
Comment (JacksonWeber) [inline src/shim/applicationinsights.ts:51]: Do we want to preserve this warning in the case the customer attempts to call setup multiple times with different connection strings?

Comment (hectorhdzg) [reply src/shim/applicationinsights.ts:51]: nothing happens if customer call this method multiple times

Comment (JacksonWeber) [reply src/shim/applicationinsights.ts:51]: Makes sense. My thought was just that we don't want to give the impression that calling setup multiple times allows for usage of multiple clients, however you're right that we need to worry about being able to setup multiple clients at all first.

File: src/shared/util/configHelper.ts [hunk 2]
```diff
@@ -77,42 +77,20 @@ export function setAutoCollectConsole(options: ApplicationInsightsOptions, value
...
-    options.logInstrumentations = {
-        ...options.logInstrumentations,
+    options.logInstrumentationOptions = {
+        ...options.logInstrumentationOptions,
         winston: { enabled: value },
         bunyan: { enabled: value },
     }
 }
 
 export function enableAutoCollectConsole(options: ApplicationInsightsOptions, value: boolean) {
```
Comment (JacksonWeber) [inline src/shared/util/configHelper.ts:87]: Doesn't look like this function is necessary anymore as `enableAutoCollectConsole()` runs inside of the config.ts file.

Comment (JacksonWeber) [reply src/shared/util/configHelper.ts:87]: As a follow up if have all of these functions used/written in the config.ts file we may be able to eliminate this helper file.

**Review by JacksonWeber — APPROVED** [review]
LGTM

**Pull request merged**